src/UI/Art/SpriteArtist/CommanderOverlayArtist.java

```diff
@@ -151,17 +151,17 @@ private static int getAnimIndex()
   /**
    * Generate an ability-power bar for the given Commander at 1x size. The requester is responsible for applying any scale factors.
    */
-  public static BufferedImage buildCoPowerBar(Commander co, double[] abilityPoints, double currentPower, boolean leftSide)
+  public static BufferedImage buildCoPowerBar(Commander co, int[] abilityPoints, int currentPower, boolean leftSide)
   {
     if( 0 == abilityPoints.length )
       return SpriteLibrary.createTransparentSprite(1, 1);
 
-    final double pixelsPerPowerUnit = 3.0;
+    final double pixelsPerPowerUnit = 3.0 / Commander.CHARGERATIO_FUNDS;
```

Comment:
Am I missing something, or are we calculating 3/9000==0 pixels per power unit? I mean, it still draws, so clearly not, but still.

Reply:
3.0 is a double value, so...

Reply:
Is each "power unit" a fund now? It used to be a star equivalent, but that clearly isn't what you're doing here.

Reply:
Yes. I did mention that on another thread, I think.
I didn't make the interface as obvious as e.g. HP since I was trying to limit how much I touched (oops?).
This does mean "1 star of charge" can always be 1 star as the carts define it, though, since the scaling is underneath that level (in funds).

Reply:
Were we defining power units differently than stars before? I know we do scaling differently, but otherwise it was just a naming difference AFAIK.

Reply:
Sort of.
The power costs were defined in *base* stars, so scaling either had to bring you to a non-integer cost or increase by an integer number of stars... and there was no way to easily refer to "stars as scaled with fatigue", while now it wouldn't be too hard.